Dockerfile

```diff
@@ -9,12 +9,24 @@ RUN \
     lsblk \
     dmidecode \
     util-linux \
-    lm-sensors \
-    speedtest-cli &&\
+    lm-sensors &&\
   apk --no-cache add \
     raspberrypi \
     || true
 
+ARG TARGETPLATFORM
+RUN \
+  if [ "$TARGETPLATFORM" = "linux/amd64" ] ; \
+    then wget -qO- https://install.speedtest.net/app/cli/ookla-speedtest-1.1.1-linux-x86_64.tgz \
+    | tar xmoz -C /bin speedtest ; \
+  elif [ "$TARGETPLATFORM" = "linux/arm64" ] ; \
+    then wget -qO- https://install.speedtest.net/app/cli/ookla-speedtest-1.1.1-linux-aarch64.tgz \
+    | tar xmoz -C /bin speedtest ; \
+  elif [ "$TARGETPLATFORM" = "linux/arm/v7" ] ; \
+    then wget -qO- https://install.speedtest.net/app/cli/ookla-speedtest-1.1.1-linux-armel.tgz \
+    | tar xmoz -C /bin speedtest ; \
+  fi
```

Comment:
A switch might seem more appropriate here, what do you think? (e.g. https://stackoverflow.com/a/68091588/9150652)

Reply:
Sure, that will be better.